test/unit/spec/PasswordReset_spec.js

```diff
@@ -12,31 +13,43 @@ define([
   'LoginRouter',
   'sandbox',
   'helpers/xhr/PASSWORD_RESET',
-  'helpers/xhr/PASSWORD_RESET_withMinComplexity',
-  'helpers/xhr/PASSWORD_RESET_withMaxComplexity',
+  'helpers/xhr/PASSWORD_RESET_withComplexity',
   'helpers/xhr/PASSWORD_RESET_error',
   'helpers/xhr/200',
   'helpers/xhr/SUCCESS'
 ],
 function (Q, _, $, OktaAuth, LoginUtil, Util, PasswordResetForm, Beacon, Expect, Router,
-          $sandbox, resPasswordReset, resPasswordResetWithMinComplexity, resPasswordResetWithMaxComplexity,
-          resError, res200, resSuccess) {
+          $sandbox, resPasswordReset, resPasswordResetWithComplexity, resError, res200, resSuccess) {
 
   var itp = Expect.itp;
   var tick = Expect.tick;
   var processCredsSpy = jasmine.createSpy();
 
+  function deepClone(res) {
+    return JSON.parse(JSON.stringify(res));
+  }
+
   function setup(settings) {
     var passwordResetResponse = resPasswordReset;
-    if(settings) {
-      switch (settings.policyComplexity) {
-      case 'min':
-        passwordResetResponse = resPasswordResetWithMinComplexity;
-        break;
+    var policyMaxComplexityDefaults = {
```

Comment:
nit: can we name this ```policyComplexityDefaults```? It contains all properties, but not really a max of any value.